pyfixest/utils/utils.py

```diff
@@ -108,13 +108,8 @@ def get_ssc(
     cluster_adj_value = 1.0
     adj_value = 1.0
 
-    if vcov_type == "hetero":
-        adj_value = N / (N - k) if adj else N / (N - 1)
-    elif vcov_type in ["iid", "CRV"]:
-        if adj:
-            adj_value = (N - 1) / (N - k)
-    else:
-        raise ValueError("vcov_type must be either iid, hetero or CRV.")
+    if adj:
+        adj_value = (N - 1) / (N - k)
```

Comment:
Is there a reference for this that you can put in the doc strings for the non-experts :D

Reply:
Here is the very detailed and excellent treatment of what fixest does: [link](https://cran.r-project.org/web/packages/fixest/vignettes/standard_errors.html) =D Though it explicitly states that it uses `(N-1) / (N-k)`, I was under the assumption it would do something different ... not sure where this came from. I will add it to the docs. Generally I think that all of these things should be documented more prominently!

Reply:
Thanks! At least adding this to the docstrings is a place to start :)

Reply:
Added a description + URL to the docstrings. Don't think it's perfect yet, will revisit tomorrow =)